### ISSUE TITLE
 want move activity state to STOPPED, ActivityScenario#moveToState(STOPPED)

### ISSUE THREAD

**Issue by LitterSun** [android/android-test#768]
### Description
I want move activity state to STOPPED，but I can not find STOPPED State enum. And I read the source code
```
public ActivityScenario<A> moveToState(State newState) {
...
            switch(newState) {
            case CREATED:
                this.activityInvoker.stopActivity(currentState.activity);
                break;
...
```
I should call moveToState(CREATED), it's hard to understand.

**Comment by TWiStErRob** [comment]
I agree, the documentation also uses "stopped" as the terminology in many places:
https://developer.android.com/guide/components/activities/activity-lifecycle#asem

There would be a way to enable this without changing the enum (much). If the `Lifecycle.State` enum was defined in Kotlin like so:
```kotlin
interface Lifecycle {
	enum class State {
		DESTROYED,
		INITIALIZED,
		CREATED,
		STARTED,
		RESUMED;
		companion object // very important, and this is why it needs to be Kotlin.
	}
}
```
Then we or `-ktx` could define this: `val Lifecycle.State.Companion.STOPPED get() = Lifecycle.State.CREATED` (or even put the aliases directly into the companion object as non-extension constants), and then - at least from Kotlin - it would be possible to do `scenario.moveToState(Lifecycle.State.STOPPED)`. Currently the only workaround we have is a top level constant or named methods.
```kotlin
val STOPPED = Lifecycle.State.CREATED
// or
inline fun <reified F : Fragment> FragmentScenario<F>.moveToStopped() = moveToState(Lifecycle.State.CREATED)
```
Would you consider named extension functions? (e.g. in fragment-testing-ktx)
